### DISCUSSION TITLE
Stuck after aborting upload process

### DISCUSSION THREAD

**Discussion by EdyTheCow** [simulot/immich-go#269] [Q&A] [answered]
I am in progress of uploading a google takeout that totals in over 500GB compressed (.zip) with around 13k images and videos. Since my internet isn't the fastest I planned on running this over few days by aborting and starting the process again. 
I am currently trying to run it for the second time but it has been stuck for around 5-6 hours without uploading any new images or videos. The log does show "file duplicated in the input" and is very slowly going through the media. 

I am now wondering if it's attempting to upload every single image and video in order to check if it already exists? Or if I am doing something wrong to resume the process from where it left?

**Comment by simulot** [comment]
> I am now wondering if it's attempting to upload every single image and video in order to check if it already exists?

It depends. Immich-go try to detect duplicates using the file name and the date of capture. This give the "file duplicated in the input". 

But if you have copied files and renamed them, this detection fails, and then it try to upload them. Then, the immich server check if it has a file with the same content, whatever its name. If it have it, the file is rejected. This give "server has same photo".

> Or if I am doing something wrong to resume the process from where it left?

You can brake the program at any point. At the next run, after having retrieved the full list of server's files (could be long), the files already uploaded with be skipped, and counted as "server has same photo"

**Reply by EdyTheCow** [reply, marked as answer]
Found the issue, seems like the account I was attempting to upload media to had a set limit of 10GB which I forgot to increase after testing my immich installation. If possible, I would suggest adding a check for storage limit on immich's user and output a warning if that's reached. Could be potentially helpful in case there's someone else as dumb as me! Thanks for a quick reply!

**Reply by simulot** [reply]
Glad you sorted it out.

I have to check the doc it this information is available.
I'd prefer to have an error message from immich.
